app/views/home.njk

```diff
@@ -27,6 +27,10 @@
         text: "Start now"
       }) }}
 
+      <p>
+        <a href="check-eligibility/bps">Check your eligibility</a>
+      </p>
```

Comment:
This looks a bit odd on the start page.  Suggest either adding a useful links section and adding a subsection as described in the start page pattern here -> https://design-system.service.gov.uk/patterns/start-pages/

Or moving it down a bit to the `Before you start` section.  If this option then needs the `govuk-body` class so it gets the correct font.